dask/dot.py

```diff
@@ -104,7 +104,7 @@ def label(x, cache=None):
     return s
 
 
-def box_label(key):
+def box_label(key, verbose=True):
```

Comment:
Any reason you made the default `verbose=True` here and False above? I think it doesn't matter much, since we always pass verbose, but maybe align them.

Reply:
Yeah, agree. It should be False.